prow/test/integration/setup-prow-components.sh

```diff
@@ -172,9 +172,33 @@ EOF
     --prow-images-file="${prowimagebuilder_yaml}" \
     --push
   set +x
+
+  build_extra_images
   log "Finished building images"
 }
 
+function build_extra_images() {
+  log "Building extra images"
+
+  build_clonerefs_ssl_disabled
+}
+
+function build_clonerefs_ssl_disabled() {
+  echo >&2 "Building clonerefs-ssl-disabled"
+  local src
+  local dest
+  src="localhost:${LOCAL_DOCKER_REGISTRY_PORT}/clonerefs:latest"
+  dest="localhost:${LOCAL_DOCKER_REGISTRY_PORT}/clonerefs-ssl-disabled:latest"
+
+  docker build --tag "${dest}" - <<EOF
+FROM ${src}
+# Allow Git to accept traffic from HTTPS servers with self-signed certs.
+ENV GIT_SSL_NO_VERIFY 1
```

Comment:
is it not possible to set ENV with ko?

Reply:
It is, but then I think the rules inside our .ko.yaml get a little messy because AFAICS the environment variable customization rules are tied to the import path, which is still `prow/cmd/clonerefs` (which we don't want to change). I don't know how it is possible to have 2 separate rules for the same import path. LMK if this is possible.

Reply:
I see the point, couldn't find a way to define two separate rules for the same import path either, so lgtm